src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -244,6 +245,173 @@ function RemoteFunctions(experimental) {
                 return;
             }
 
+            var setVisibility = function (el){
+                if (
+                    !config.remoteHighlight.showPaddingMargin || 
+                    parseInt(el.height, 10) <= 0 || 
+                    parseInt(el.width, 10) <= 0 
+                ) {
+                    el.display = 'none';
+                } else {
+                    el.display = 'block';
+                }
+            };
+            
+            var elementStyling = window.getComputedStyle(element, null);
+            
+            var calculateSize = function () {
+                var sum = 0;
+                var i;
+                for (i = 0; i < arguments.length; i++) {
```

Comment:
Not a huge one of using `arguments`, can we instead pass an array and loop through that (with `Array.prototype.reduce` for example?)

```
var calculateSize = function (values) {
  return values.reduce(function (val) {
    return parseInt(val, 10);
  }, 0);
};
```

(Side note, I wish we could use ES2015 so this would be just `const calculateSize = (values) => values.reduce(val => parseInt(val, 10), 0)` 😸)